apps/moderatorfeedback/templates/meinberlin_moderatorfeedback/moderatorfeedback.html

```diff
@@ -0,0 +1,7 @@
+{% if item.get_moderator_feedback %}
+    {% with item.get_moderator_feedback as moderator_feedback %}
+    <span class="moderator-feedback moderator-feedback--{{ moderator_feedback.value_class }}">
+        {% if moderator_feedback.value %}{{ moderator_feedback.display }}{% endif %}
+    </span>
+    {% endwith %}
+{% endif %}
```

Comment:
Better add an `include` subfolder to indicate that this is not a standalone template.